docs/pages/steps/jira_add_comment.md

```diff
@@ -10,10 +10,7 @@ folder: steps
 
```

Comment:
[updated steps docs](https://github.com/jenkinsci/jira-steps-plugin/pull/39/commits/b568b4bf98e599221b74a3fcedabeae493e4025e) saying that we support all fields that your customized api provides and about using the fields from response returned.